watertap/flowsheets/full_water_resource_recovery_facility/test/test_BSM2_P_extension.py

```diff
@@ -234,3 +248,13 @@ def test_costing(self, system_frame):
         assert value(m.fs.costing.total_operating_cost) == pytest.approx(
             836020.408, rel=1e-3
         )
+
+    @pytest.mark.component
+    def test_condition_number(self, system_frame):
```

Comment:
I am pretty sure this is the case but will ask anyway- we can't mix and match application of the "old" scaling with the "new" scaling within a flowsheet, right? That is, you can't apply the ADscaler to the AD and calculate_scaling_factors everywhere else, right?  If you could, then we could apply the new scaler class to the AD and see its impact on the flowsheet condition number.

Reply:
I forget the logic, but I recall Andrew mentioning that we should not mix and match the scaling tools. That being said, I can look into this when I have the time just to be sure.